backoff.go

```diff
@@ -11,9 +11,9 @@ type BackOff func(currentRetryCount int) time.Duration
 // ExponentialBackOff computes an exponential back-off
 func ExponentialBackOff(minTimeout time.Duration) BackOff {
 	return func(currentRetryCount int) time.Duration {
-		jitter := rand.Float64()
+		jitter := rand.Float64() + 0.5
 		strategy := 1 << currentRetryCount
-		backoff := (1 + float64(strategy)*jitter) * minTimeout.Seconds() * float64(time.Second)
+		backoff := jitter * float64(strategy) * float64(minTimeout.Nanoseconds())
```

Comment:
This way `backoff` can be up to 50% less than `minTimeout`, which contradicts to the name of the argument. Is that what we want?